packages/dapp/src/components/utility/Tabs.tsx

```diff
@@ -1,17 +1,32 @@
 import * as React from "react";
 import styled from "styled-components";
 
-const StyledUL = styled.ul`
-  border-bottom: 1px solid #aaa;
-  margin: 0 0 10px;
-  padding: 0;
+const TabContainer = styled.div`
+  display: flex;
+  justify-content: center;
+  margin: 0 auto;
 `;
 
+const TabNav = styled.div`
+  margin: 0 auto 50px;
+  &.listings-nav {
+    background-color: #e9e9ea;
```

Comment:
Is this color not in `@joincivil/components`? 

If it isn't, maybre we should use the closest grey that we have defined.